client/client.go

```diff
@@ -7,13 +7,13 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
-// ClientFactoryInterface interface for the client factory
-type ClientFactoryInterface interface {
-	New(url string) ClientInterface
+// IClientFactory interface for the client factory
+type IClientFactory interface {
+	New(url string) IClient
 }
 
-// ClientInterface is the interface that defines the implementation of all the endpoints
-type ClientInterface interface {
+// IClient is the interface that defines the implementation of all the endpoints
+type IClient interface {
```

Comment:
Saw that in `types/interfaces/interfaces.go` you are using different naming convention for interfaces. Can you unify those? Either all of them should be called `ISomething` or just `Something` (and the specific implementation `SomethingImpl`), so that we are consistent through entire codebase. WDYT?

Reply:
Fix: https://github.com/0xPolygon/cdk-data-availability/pull/34/commits/8bfd3660a4876e9a4631498e1baaa18fc5a18119